crankshaft-engine/src/service/runner/backend/tes.rs

```diff
@@ -56,8 +56,91 @@ impl crate::Backend for Backend {
+                                //     .flat_map(|task| task.logs)
+                                //     .map(|log| {
+                                //         let status =
+                                //             log.exit_code.expect("exit code to be present");
+
+                                //         #[cfg(unix)]
+                                //         let output = Output {
+                                //             status: ExitStatus::from_raw(status as i32),
+                                //             stdout: log
+                                //                 .stdout
+                                //                 .unwrap_or_default()
+                                //                 .as_bytes()
+                                //                 .to_vec(),
+                                //             stderr: log
+                                //                 .stderr
+                                //                 .unwrap_or_default()
+                                //                 .as_bytes()
+                                //                 .to_vec(),
+                                //         };
+
+                                //         #[cfg(windows)]
+                                //         let output = Output {
+                                //             status: ExitStatus::from_raw(status),
+                                //             stdout: log
+                                //                 .stdout
+                                //                 .unwrap_or_default()
+                                //                 .as_bytes()
+                                //                 .to_vec(),
+                                //             stderr: log
+                                //                 .stderr
+                                //                 .unwrap_or_default()
+                                //                 .as_bytes()
+                                //                 .to_vec(),
+                                //         };
+
+                                //         output
+                                //     });
+
+                                // let mut outputs = NonEmpty::new(results.next().unwrap());
+                                // outputs.extend(results);
+                                let outputs = NonEmpty::new(Output {
+                                    status: ExitStatus::from_raw(0),
+                                    stdout: Vec::new(),
+                                    stderr: Vec::new(),
+                                });
+
+                                return Ok(outputs);
+                            } else {
+                                debug!("Task was NOT completed for {task_id}. Looping...");
+                            }
+                        } else {
+                            debug!("State was NOT set for {task_id}. Looping...");
```

Comment:
Can we downcase the first letter in all of these logs? I suppose this was me, but I think we should keep with our convention.

Reply:
Perhaps just do a quick check of all of these, not just the ones in this file.

Reply:
The only ones I could find were in this file.